client/my-sites/plans/current-plan/jetpack-checklist/header.js

```diff
@@ -20,13 +20,21 @@ const JetpackChecklistHeader = ( { isPaidPlan, translate } ) => (
 		/>
 		<div className="jetpack-checklist__header-content">
 			<CardHeading>
-				{ translate( "Let's start by securing your site with a few essential security features" ) }
+				{ isCompleted ? (
+					<>
+						{ translate( 'Congratulations' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **76** times:
`translate( 'Congratulations!' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).